src/components/Imprint/ImprintContent.tsx

```diff
@@ -0,0 +1,54 @@
+import { DefaultTypography as Typography, DefaultLink as Link } from '@common/components'
+import { useTranslation } from 'react-i18next'
+export const ImprintContent = () => {
+  const { t } = useTranslation()
+  return (
+    <div>
+      <Typography variant="h4">{t('components.ImprintContent.Title')}</Typography>
+      <p></p>
```

Comment:
MUI statt plain html wegen styling 
```suggestion
      <Typography paragraph variant="body2"/>
```